contents/handbook/growth/marketing/newsletter.md

```diff
@@ -4,24 +4,75 @@ sidebar: Handbook
+
+Each month, Joe combines new orgs into [the startups cohort](https://app.posthog.com/cohorts/44794) and exports this to Customer.io for this email. 
+
+The email is usually comprised of three sections, which inform users of new guides which are relevant to startup use-cases, new betas which are available for them to try, and a spotlight written of a new org in the program. We end by asking for feedback.
+
+We categorize these emails as `Actually useful marketing emails` in Customer.io, so users can unsubscribe if they wish. 
+
+### Onboarding emails
+**Owner:** Joe
+
+The onboarding email logic is complex, and regularly changes as we test new ideas. Any changes to it are, as with all other email campaigns, documented in [the Meta repo](https://github.com/PostHog/meta/). The latest revision is [Onboarding 3.1](https://github.com/PostHog/meta/issues/127). 
+
+The onboarding flow is composed of several workflows, with dedicated flows for: 
+
+- All cloud sign-ups
+- All self-hosted sign-ups
+- All open-source deploys
+- Cloud users who have not subscribed to replays
+- Cloud users who have not subscribed to feature flags
+
+We aim for all content in these flows to be relevant and helpful to users, without being salesy. We use a mix of personal and 'designed' email styles, with personal emails coming directly from the inbox of relevant people (namely, Andy and Joe). When we get replies or feedback to these emails, we share it on Slack. 
+
+A full description of the flows above is found in [the Onboarding 3.0 issue](https://github.com/PostHog/meta/issues/123). 
+
+We tag all these email flows as `onboarding` in Customer.io and categorize them as `Welcome emails` so that users can easily manage their preferences. 
+
+### Other customer communications
+**Owner:** Joe
+
+Any ad-hoc customer communications over email are owned by the marketing team, and are usually sent via Customer.io. These can include product updates, outage alerts, or other PostHog news if needed.
+
+These emails are usually tagged as `Service updates` in Customer.io when they include important account or product information. These emails are given a dedicated unsubscribe option in the footer, making it clear that we do not recommend users unsubscribe to these emails. 
+
+Important service updates are the _only_ type of email we may send to unsubscribed users, and only if we feel it is warranted to do so.
+
+> `Service updates` emails are usually part of an [engineering incident](/handbook/engineering/incidents.md). If an incident is declared, please ensure the marketing team is aware as soon as possible by posting in the team-marketing Slack.
+
+We are currently testing email as a channel for other activities, such as product upsells. We tag these emails as `Actually useful marketing emails` in Customer.io and have found the a personal, non-designed approach works best. 
```

Comment:
Could this be considered onboarding? Not onboarding into PostHog, but into new products?

Reply:
It could, but it's just tests for now and they don't run as consistent campaigns.